manifests/cf-manifest/manifest/020-cf-properties.yml

```diff
@@ -452,9 +452,11 @@ properties:
       client_secret: (( grab secrets.uaa_cc_client_secret ))
 
     jwt:
-      verification_key: (( grab secrets.uaa_jwt_verification_key ))
```

Comment:
If this secret is no longer used, can we remove it from the secret generator and anywhere else it's referenced.

Reply:
Good call. I'll update this PR to do that, if indeed it's unused.

Reply:
I think it is generated here, by extracting the public key from the cert: https://github.com/alphagov/paas-cf/blob/f24c36623d0b72be26c606f74e8248c6a8f1c966/concourse/pipelines/create-cloudfoundry.yml#L549-L550

But maybe is being used in datadog.

Reply:
I couldn't find any other references to it in this repo...